src/include/batch_request.h

```diff
@@ -289,7 +301,7 @@ struct batch_request {
 		struct rq_jobfile rq_jobfile;
 		char rq_rdytocommit[PBS_MAXSVRJOBID + 1];
 		char rq_commit[PBS_MAXSVRJOBID + 1];
-		struct rq_manage rq_delete;
+		struct rq_deljob rq_delete;
```

Comment:
This is bit confusing as you are using same structure for both deljob and deljoblist. I would suggest to keep rq_manage as it for deljob and use new struct (rename it to rq_deljoblist) for deljoblist

Reply:
@bremanandjk  did you miss this comment?